src/main/java/TaskList.java

```diff
@@ -0,0 +1,178 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to handle all the tasks.
+ */
+public class TaskList {
+    private static final String ADD_TODO_COMMAND = "todo";
+    private static final String ADD_DEADLINE_COMMAND = "deadline";
+    private static final String ADD_EVENT_COMMAND = "event";
+
+    List<Task> lst = new ArrayList<>();
```

Comment:
Is this `lst` variable meant to be accessed outside this class? If not, perhaps a getter method would be more appropriate rather than giving direct access to the other classes, which would result in undesirable side effects.